atom/common/native_mate_converters/net_converter.cc

```diff
@@ -46,6 +46,12 @@ v8::Local<v8::Value> Converter<scoped_refptr<net::X509Certificate>>::ToV8(
                                    encoded_data.size()).ToLocalChecked();
   dict.Set("data", buffer);
   dict.Set("issuerName", val->issuer().GetDisplayName());
+  dict.Set("subjectName", val->subject().GetDisplayName());
+  dict.Set("serialNum", val->serial_number());
```

Comment:
this looks good, but any special reason we are abbreviating this? If it matches up to something else that's great, but otherwise it seems like it could lead to unexpected null results